### ISSUE TITLE
Project Name Link Color Contrast

### ISSUE THREAD

**Issue by nidhi-k3** [aewindle110/DroneWQ#37]
Project name links have insufficient color contrast (3.15:1). Text color #3498db on white background doesn't meet WCAG 2 AA standards.

Locations:
"Algae Survey" link
"Coastal Water Quality" link
"Lake Michigan Study" link
"Reef Monitoring 2025" link
"Turbidity Assessment" link

Fix:
Change link color from #3498db to #0056b3 (darker blue)
